gstreamer-pbutils/src/auto/discoverer_audio_info.rs

```diff
@@ -0,0 +1,74 @@
+// This file was generated by gir (d933f9a+) from gir-files (???)
+// DO NOT EDIT
+
+use DiscovererStreamInfo;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct DiscovererAudioInfo(Object<ffi::GstDiscovererAudioInfo>): DiscovererStreamInfo;
+
+    match fn {
+        get_type => || ffi::gst_discoverer_audio_info_get_type(),
+    }
+}
+
+unsafe impl Send for DiscovererAudioInfo {}
+unsafe impl Sync for DiscovererAudioInfo {}
+
+pub trait DiscovererAudioInfoExt {
```

Comment:
You can set `trait = false` for these types as they won't have any further subclasses anymore